packages/rspack-test-tools/tests/builtinCases/plugin-javascript/import-meta/__snapshots__/output.snap.txt

```diff
@@ -0,0 +1,85 @@
+```js title=main.js
+(self["webpackChunkwebpack"] = self["webpackChunkwebpack"] || []).push([["main"], {
+"./index.js": (function (__unused_webpack_module, __unused_webpack_exports, __webpack_require__) {
+const { pathToFileURL } = __webpack_require__("url");
+const url = pathToFileURL(
+	"./tests/builtinCases/plugin-javascript/import-meta/index.js",
+).toString();
+
+const filename = "/index.js";
```

Comment:
It appears that `__filename` is handled by `crates/rspack_plugin_rstest/src/parser_plugin.rs` using a different strategy.